File: samples_generated_code/sample_2/phi4/baseline_code_py/code_row_61.py
import sys

def getValueFromArray(arr, size, idx):
    if 0 <= idx < size:
        return arr[idx]
    else:
        return -1

def main():
    n = int(input().strip())
    arr = list(map(int, input().split()))
    
    if len(arr) != n:
        sys.exit(-1)
    
    index = int(input().strip())

    result = getValueFromArray(arr, n, index)
    sys.exit(result)

if __name__ == "__main__":
    main()

File: samples_generated_code/sample_2/phi4/baseline_code_java/code_row_61.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int n = scanner.nextInt();
        
        if (n < 1 || n > 1000) { // Assuming size constraint for safety
            System.exit(1); // Exit with code 1 in case of invalid input size
        }
        
        int[] array = new int[n];
        
        for (int i = 0; i < n; i++) {
            array[i] = scanner.nextInt();
        }
        
        int index = scanner.nextInt();
        
        if (index < 0 || index >= n) {
            System.exit(1); // Exit with code 1 in case of invalid index
        }
        
        int result = getValueFromArray(array, n, index);
        System.out.println(result);
        System.exit(result);
    }

    public static int getValueFromArray(int[] array, int size, int index) {
        return array[index];
    }
}